styles.css

```diff
@@ -454,6 +454,207 @@ input[type=name] {
   align-items: center;
 }
 
+#work-section {
+  font-family: sans-serif;
+  font-weight: 500;
+  margin-top: 70px;
+}
+
+#box-1 {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  margin-top: 40px;
+}

```

Comment:
- [ ] As far as I understood this is used in each project card, if I am right then you should use a flexbox instead of grid, kindly fix this. 😁